in_toto/models/common.py

```diff
@@ -99,9 +99,10 @@ def verify_signatures(self, keys_dict):
       keyid = signature["keyid"]
       try:
         key = keys_dict[keyid]
-      except:
+      except KeyError:
```

Comment:
This method doesn't validate it's arguments, so I'm not sure where/when/how its done, but `keys_dict[keyid]` can potentially raise `TypeError` if `keyid` happens to contain an unhashable type.

```Python
>>> dict1 = {}
>>> dict1[{'foo': 1}]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unhashable type: 'dict'
```